src/lib/viewers/BaseViewer.js

```diff
@@ -1037,6 +1038,19 @@ class BaseViewer extends EventEmitter {
         this.annotator.scrollToAnnotation(data);
     }
 
+    handleAnnotationsOnLoad = () => {
+        const {
+            file: { id },
+        } = this.options;
+
+        const loadedAnnotationId = getProp(this.options, `fileOptions.${id}.annotations.activeId`, null);
+        const annotation = loadedAnnotationId ? this.annotator.getAnnotationById(loadedAnnotationId) : null;
```

Comment:
We should avoid calling methods on the annotator when possible, especially when they involve the internal state of the library.